client/README.md

```diff
@@ -44,6 +44,15 @@ $ pnpm pack-module -- module-name
 
 This will "package" the module `module-name`. What this does is that it starts a special build process on that module, which will yield a publishable NPM package in the modules `dist` folder. Any important changes that would've needed to been made to the module and its `package.json` are handled automatically.
 
+### Publishing
+
+To publish a module, run this command:
+```
+$ pnpm publish-module -- module-name
+```
+
+This will package the module, as described above, and then publish it. Of course, you want to exercise caution when publishing.
```

Comment:
I would include a more notable caution / notice sign than this. Publishing is public and not really reversible, not something to be done to test it out.